.circleci/config.yml

```diff
@@ -20,20 +20,47 @@ jobs:
           name: Sample Distribution Analyses
           command: ./scripts/run_in_ci.sh bash "analyses/sample-distribution-analysis/run-sample-distribution.sh"
 
+      # TODO: The data files for CI need to be fixed https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/527
```

Comment:
After the subset was redone, this doesn't rectify this?

Reply:
No it did not - more details in #527